packages/block-library/src/query-pagination-next/index.php

```diff
@@ -50,7 +50,8 @@ function render_block_core_query_pagination_next( $attributes, $content, $block
 		}
 		wp_reset_postdata(); // Restore original Post Data.
 	}
-	return $content;
+
+	return '<div class="wp-block-query-pagination-next">' . $content . ' ' . $attributes['arrow'] . '</div>';
```

Comment:
We should first check if `arrow` is set before adding the extra space. Also this should be inside `$label` to be in the link, no? And the wrapper seems useful to me, as I mentioned above.